### PR TITLE
Change some output messages and minor refactoring

### LINKED ISSUE #74: refactor: modify mentions of Person and AddressBook
Update the mentions of Person and AddressBook to be more consistent with our project (i.e. Task, Module, ModuleList, TaskList, etc.), in JavaDocs and method/variable names

### PR DESCRIPTION
**Pull request by HoJunHao2000**
Fix #74 

- renamed TaskbookPaser to ModtrektParser to be more accurate
- renamed all the persons/address methods and parameters to modules/task list/book
- changed error output to show format instead of example to be more concise

### CODE REVIEW

File: src/main/java/modtrekt/model/Model.java [hunk 1]
```diff
@@ -127,8 +135,8 @@ public interface Model {
 
     /**
      * Replaces the given person {@code target} with {@code editedModule}.
```
Comment (domoberzin) [inline src/main/java/modtrekt/model/Model.java:137]: theres still a mention of person here

File: src/main/java/modtrekt/MainApp.java [hunk 2]
```diff
@@ -96,11 +99,11 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             }
             moduleList = moduleListOptional.orElseGet(SampleDataUtil::getSampleModuleList);
         } catch (DataConversionException e) {
-            logger.warning("Data file not in the correct format. Will be starting with an empty AddressBook");
+            logger.warning("Data file not in the correct format. Will be starting with an empty ModuleList");
```
Comment (domoberzin) [inline src/main/java/modtrekt/MainApp.java:102]: maybe can specify that the app will be starting with both an empty module and task list

File: src/main/java/modtrekt/MainApp.java [hunk 3]
```diff
@@ -96,11 +99,11 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
...
             taskBook = new TaskBook();
             moduleList = new ModuleList();
         } catch (IOException e) {
-            logger.warning("Problem while reading from the file. Will be starting with an empty AddressBook");
+            logger.warning("Problem while reading from the file. Will be starting with an empty ModuleList");
```
Comment (domoberzin) [inline src/main/java/modtrekt/MainApp.java:106]: same here

File: src/main/java/modtrekt/logic/parser/AddCommandParser.java [hunk 4]
```diff
@@ -32,16 +32,14 @@ public Command parse(String args) throws ParseException {
...
                 CliSyntax.PREFIX_MOD_CODE, CliSyntax.PREFIX_MOD_CREDIT)) {
+
             ModName name = ParserUtil.parseName(argMultimap.getValue(CliSyntax.PREFIX_MOD_NAME).get());
             ModCode code = ParserUtil.parseCode(argMultimap.getValue(CliSyntax.PREFIX_MOD_CODE).get());
             ModCredit credit = ParserUtil.parseCredit(argMultimap.getValue(CliSyntax.PREFIX_MOD_CREDIT).get());
 
             Module module = new Module(code, name, credit);
 
             return new AddCommand(module);
-        } else if (arePrefixesPresent(argMultimap, CliSyntax.PREFIX_MODULE, CliSyntax.PREFIX_MOD_NAME)
```
Comment (domoberzin) [inline src/main/java/modtrekt/logic/parser/AddCommandParser.java:42]: Perhaps we should leave this in first, this enforces that the user has to use a specific command, so adding extra prefixes will not allow them to add the module

File: src/main/java/modtrekt/model/module/UniqueModuleList.java [hunk 5]
```diff
@@ -29,15 +29,15 @@ public class UniqueModuleList implements Iterable<Module> {
...
+     * Returns true if the list contains an equivalent module as the given argument.
      */
     public boolean contains(Module toCheck) {
         requireNonNull(toCheck);
         return internalList.stream().anyMatch(toCheck::isSameModule);
     }
 
     /**
-     * Returns true if the list contains an equivalent person as the given argument.
+     * Returns true if the list contains an equivalent module as the given argument.
```
Comment (domoberzin) [inline src/main/java/modtrekt/model/module/UniqueModuleList.java:40]: Maybe more details could be added to differentiate this from the method above, but I guess we can also leave this for when we update the documentation as a whole

File: src/main/java/modtrekt/model/module/UniqueModuleList.java [hunk 6]
```diff
@@ -47,7 +47,7 @@ public boolean containsCode(ModCode toCheck) {
...
```
Comment (domoberzin) [inline src/main/java/modtrekt/model/module/UniqueModuleList.java:50]: same here

File: src/main/java/modtrekt/model/person/NameContainsKeywordsPredicate.java [hunk 7]
```diff
@@ -1,32 +0,0 @@
...
-import java.util.List;
-import java.util.function.Predicate;
-
-import modtrekt.commons.util.StringUtil;
-import modtrekt.model.task.Task;
-
-/**
- * Tests that a {@code Person}'s {@code Name} matches any of the keywords given.
- */
-public class NameContainsKeywordsPredicate implements Predicate<Task> {
```
Comment (domoberzin) [inline src/main/java/modtrekt/model/person/NameContainsKeywordsPredicate.java:12]: Maybe we should save this file for when we do our find feature

File: src/main/java/modtrekt/ui/HelpWindow.java [hunk 8]
```diff
@@ -15,7 +15,9 @@
  */
 public class HelpWindow extends UiPart<Stage> {
 
-    public static final String USERGUIDE_URL = "https://se-education.org/addressbook-level3/UserGuide.html";
+    public static final String USERGUIDE_URL =
```
Comment (domoberzin) [inline src/main/java/modtrekt/ui/HelpWindow.java:18]: Let's modify this link to use the GitHub one!
`https://ay2223s1-cs2103t-w10-4.github.io/tp/UserGuide.html`

If you want to find this, the formatting is the same as the way we obtain our IP UG/DG etc.

**Review by domoberzin — CHANGES_REQUESTED** [review]
Overall good work, just a few small things to change (UG url etc.)

Not sure if we should delete all those test cases though, think we can recycle in future, but if anything we can just restore the files so I think its fine!

Good work!

**Comment by HoJunHao2000** [comment]
> Overall good work, just a few small things to change (UG url etc.)
> 
> Not sure if we should delete all those test cases though, think we can recycle in future, but if anything we can just restore the files so I think its fine!
> 
> Good work!

Alright! ill make all these changes in #79 as that branch branches from this branch

**Pull request merged**